### DISCUSSION TITLE
How to view who deleted an asset

### DISCUSSION THREAD

**Discussion by ColinSadler** [snipe/snipe-it#11476] [Q&A]
Is there anyways to see who deleted an asset?

**Comment by snipe** [comment]
That should be in the asset history

**Reply by ColinSadler** [reply]
So it shows up as an "update" under action with everything changed to NA.  That makes sense, thanks!

**Reply by snipe** [reply]
It shouldn't tho - we have a delete action. 

<img width="1575" alt="Screen Shot 2022-07-07 at 12 45 27 PM" src="https://user-images.githubusercontent.com/197404/177859185-5541d4ab-4c9b-4eff-91ee-66d6acd8a4f3.png">

**Reply by ColinSadler** [reply]
Interesting it shows up for me this way
![Capture](https://user-images.githubusercontent.com/108884848/177860338-a0c07f08-af50-404e-9f17-9f4c14e4aa45.JPG)
